toolkit.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+import sys
+import cv2 as cv
+import numpy as np
+import ntpath
+import functools
+import time
+
+
+def timeit(func):
+    """
+    Poors man profiling function to
+    measure time it takes to finish function.
+    """
+    @functools.wraps(func)
+    def newfunc(*args, **kwargs):
+        start_time = time.time()
+        out = func(*args, **kwargs)
+        elapsed_time = time.time() - start_time
+        msg = 'function [{}] finished in {} ms'
+        print(msg.format(func.__name__, int(elapsed_time * 1000)))
+        return out
+    return newfunc
+
+
+class BoardToolkit:
+    def __init__(self, lo_thresh=100, hi_thresh=255):
+        self.lo_thresh = lo_thresh
+        self.hi_thresh = hi_thresh
+
+    def __get_index(self, arr):
+        """
+        Function for returning min, max tuples from
```

Comment:
@Foxiy If you want to get serious with docstrings they should also describe the input and output parameters. I personally do not care about the format that much but I found the [Google one](https://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html) to be pretty reasonable. If you do not want to support two oxygens there are also a [couple of others](https://stackoverflow.com/questions/5334531/python-documentation-standard-for-docstring) which I find way less readable.

That being said, what you have is both difficult to read and tough to comprehend. I believe that standardization might be of some help.